lua/nvim-treesitter/install.lua

```diff
@@ -267,11 +265,7 @@ function M.uninstall(lang)
       M.uninstall(lang)
     end
   elseif lang then
-    local package_path, err = utils.get_package_path()
-    if err then
-      print(err)
-      return
-    end
+    local package_path = utils.get_package_path()
```

Comment:
I removed the error handling since this should only fail under very exceptional circumstances now, but let me know if you prefer keeping it for consistency.